app/policies/admin/comment_policy.rb

```diff
@@ -0,0 +1,22 @@
+# Pundit policy for administration of comments
+class Admin::CommentPolicy < Admin::DefaultPolicy
+  def hide_comment?
```

Comment:
The `_comment` suffixes feel a little unnecessary - consider removing them from the `CommentPolicy` and specifying the method name in the controller `authorize` calls.